conda/third_party/dali_opencv/recipe/meta.yaml

```diff
@@ -19,8 +19,8 @@
     version: {{ build_version }}
 
   source:
-    fn: opencv-4.9.0.tar.gz
-    url: https://github.com/opencv/opencv/archive/refs/tags/4.9.0.tar.gz
+    fn: opencv-4.10.0.tar.gz
+    url: https://github.com/opencv/opencv/archive/refs/tags/4.10.0.tar.gz
```

Comment:
I think you need to adjust the sha digest as well.

And what about libjpeg-turbo?

Reply:
Done